### PR TITLE
Update android-installation.mdx

### PR DESCRIPTION
**Pull request by SuQiang2007**

### CODE REVIEW

File: data/android/android-installation.mdx [hunk 1]
```diff
@@ -25,6 +25,103 @@ dependencies {
 }
 ```
 
+4. Config CustomTab!
+If you want to use CustomTab to show content to users(Recommand), you need to config the following contents on your AndroidManifest.xml:
```
Comment (ChuckD30) [inline data/android/android-installation.mdx:29]:
```suggestion
If you want to use CustomTab to show content to users (Recommended), you need to configure the following on your AndroidManifest.xml:
```

File: data/android/android-installation.mdx [hunk 2]
```diff
@@ -25,6 +25,103 @@ dependencies {
...
+    <intent-filter>
+        <action android:name="android.intent.action.VIEW"/>
+        <category android:name="android.intent.category.DEFAULT"/>
+        <category android:name="android.intent.category.BROWSABLE"/>
+        <data android:scheme="mwsdk"/>
+    </intent-filter>
+</activity>
+```
+
+So, the finnaly file would looked as this:
```
Comment (ChuckD30) [inline data/android/android-installation.mdx:51]:
```suggestion
The final file would look like this:
```

File: data/android/android-installation.mdx [hunk 3]
```diff
@@ -25,6 +25,103 @@ dependencies {
...
+                <category android:name="android.intent.category.BROWSABLE"/>
+                <data android:scheme="mwsdk"/>
+            </intent-filter>
+        </activity>
+    </application>
+
+</manifest>
+```
+
+**Add dependence**
```
Comment (ChuckD30) [inline data/android/android-installation.mdx:99]:
```suggestion
**Add dependency**
```

File: data/android/android-installation.mdx [hunk 4]
```diff
@@ -25,6 +25,103 @@ dependencies {
...
+```xml
+implementation 'androidx.browser:browser:1.4.0'
+```
+
+And then, you will see CustomTab when you try to open our prepared web page.
```
Comment (ChuckD30) [inline data/android/android-installation.mdx:104]:
```suggestion
You should now see CustomTab when you try to open our prepared web page.
```

File: data/android/android-installation.mdx [hunk 5]
```diff
@@ -25,6 +25,103 @@ dependencies {
...
+
+
+>*Tips:If you want to make your app has only one task, you can set your own activity to SingTast:*
```
Comment (ChuckD30) [inline data/android/android-installation.mdx:107]:
```suggestion
>*Tips: If you want your app to have only one task, you can set your own activity to SingTast:*
```

File: data/android/android-installation.mdx [hunk 6]
```diff
@@ -35,17 +132,16 @@ Activity activity = this;
...
-MirrorSDK.getInstance().SetApiKey(apiKey);
-MirrorSDK.getInstance().SetDebug(useDebugMode);
-MirrorSDK.getInstance().InitSDK(activity,MirrorEnv.DevNet);
+MirrorWorld.setDebug(useDebugMode);
+MirrorWorld.initMirrorWorld(activity,apiKey,MirrorEnv.DevNet);
 ```
 
 ### Guide user to login
 Users need to log in to use some MirrorSDK features (or APIs).
 If you want them to login(or again), you can use the following code:
```
Comment (ChuckD30) [inline data/android/android-installation.mdx:141]:
```suggestion
If you want them to login for the first time or once again, use the following code:
```

File: data/android/android-installation.mdx [hunk 7]
```diff
@@ -74,7 +170,7 @@ MirrorSDK.getInstance().CheckAuthenticated(emailAddr, new BoolListener() {
 Users may want to check their wallet in the app, you can use openWallet API:
```
Comment (ChuckD30) [inline data/android/android-installation.mdx:170]:
```suggestion
Users may want to check their wallet in the app, you can use the `openWallet` API for this:
```

**Review by ChuckD30 — CHANGES_REQUESTED** [review]
hey @SuQiang2007 I've suggested and requested the following changes. Thanks

**Comment by SuQiang2007** [comment]
Have complete all the comments.

**Pull request merged**